examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/README.md

```diff
@@ -30,9 +29,16 @@ python run_clm.py \
   --tune \
   --output_dir /path/to/checkpoint/dir
 ```
+## Language-Modeling Task
```

Comment:
Let's remove this `run_tuning.sh` method, it's duplicate with `python run_clm.py`. It doesn't make sense for users without model download path.